WordPress/UITests/Tests/DashboardTests.swift

```diff
@@ -0,0 +1,28 @@
+import UITestsFoundation
+import XCTest
+
+class DashboardTests: XCTestCase {
+
+    override func setUpWithError() throws {
+        setUpTestSuite()
+
+        try LoginFlow.login(
+            siteUrl: WPUITestCredentials.testWPcomSiteAddress,
+            email: WPUITestCredentials.testWPcomUserEmail,
+            password: WPUITestCredentials.testWPcomPassword
+        )
+    }
+
+    override func tearDownWithError() throws {
+        takeScreenshotOfFailedTest()
+        removeApp()
+    }
+
+    func testDomainsCardNavigation() throws {
```

Comment:
i ran the test locally on the WP test plan and noticed that it failed, which i think is expected? (is this feature Jetpack only?) perhaps it's time we add some identifiers on the tests to know which are JP-specific and WP-specific? (and to disable the test that shouldn't run on the test plan level)

Reply:
Good point. FWIW, I tried selecting `WordPressUITests` target yesterday for execution, for the same reason - to exclude the test form there - and failed to have the target tests enabled. I assumed that the WP tests were all disabled (and also they do not run on CI), but I was wrong, apparently. 

I could do this now, though, it looks like the process is now different after WP/JP separation.

I excluded the test from `WordPressUITests.xctestplan` and added a comment above the test with 71dbf793c6d5b9cd73cf5564fb6c71df0046a4c2

Reply:
> is this feature Jetpack only?

Yes, it's JP only 👍

Reply:
I'm not sure if this separation of UI tests was discussed before. I haven't worked much on UI tests.

Theoretically we could make a distinction between Jetpack and WordPress tests even more clearly:

- Add them to separate `JetpackUITests` folder
- Add them to only to `JetpackUITests` target
- Name differently (?), e.g `JetpackDashboardTests`, although it may be unecessary

<img width="229" alt="image" src="https://user-images.githubusercontent.com/4062343/233369746-00b98bc8-111e-4cf8-a30f-cbebc73edc56.png">

What are your thoughts on that? What would be the most comfortable way?

Reply:
> ...and from my POV, this is quite a comfortable way to manage this

Got it! Test plans definitely make sense! 👍